tests/data/aos7.json

```diff
@@ -79023,7 +79023,7 @@
                     "designatedRoot": "2cfaa24ab00b",
                     "designatedCost": 1900,
                     "designatedBridge": "2cfaa2495c25",
-                    "designatedPort": 11,
+                    "designatedPort": 0,
```

Comment:
Is this one a breakage?

Reply:
It's because the input is not valid hex. 
The input is like `bq` so b=11 q=error
Not sure what is correct here.